client.go

```diff
@@ -1375,3 +1398,455 @@ func (c *ToznySDKV3) UnbrokerShare(ctx context.Context, authorizerClientID strin
 		RecordType:   recordType,
 	})
 }
+
+type SecretRequest struct {
+	SecretType  string
+	SecretName  string
+	SecretValue string
+	Description string
+	FileName    string
+}
+
+// CreateSecret makes a secret of the specified type and shares it with the client
+func (c *ToznySDKV3) CreateSecret(ctx context.Context, login TozIDLoginRequest, secret SecretRequest) (*pdsClient.Record, string, error) {
```

Comment:
method signature wise,
```
func (c *ToznySDKV3) CreateSecret(ctx context.Context, secret CreateSecretRequest) (Secret, error) {
```


and type for secret should be something like

```
type Secret struct {
SecretType  string
 	SecretName  string
 	SecretValue string
 	Description string
 	FileName    string
        Record  pdsClient.Record,
}
```

if we want, for today there can just be the Record field, but the point is that Secrets is an abstraction over Records, and functionally if I pass a secret into an sdk method, I expect to get a secret back.

The realm name (which appears to be the only use for login in this method) should be a field in the `CreateSecretRequest` struct